### PR TITLE
Update label names in keylabeler.yml

### PR DESCRIPTION
**Pull request by DrSmugleaf**
Added in https://github.com/space-wizards/space-station-14/pull/2956

### CODE REVIEW

File: .github/keylabeler.yml [hunk 1]
```diff
@@ -9,21 +9,14 @@ caseSensitive: false
...
-    "[EVENT]": "Feature: Events"
-    "[INTERACTION]": "Feature: Interaction"
-    "[MEDICAL]": "Feature: Medical"
-    "[PHYSICS]": "Feature: Physics"
-    "[POWER]": "Feature: Power"
-    "[SPRITES]": "Feature: Sprites"
-    "[SPRITE]": "Feature: Sprites"
-    "[UI]": "Feature: UI"
+    "[ATMOS]": "Area: Atmos"
+    "[BODY]": "Area: Body System"
```
Comment (Visne) [inline .github/keylabeler.yml:13]: `[BODY SYSTEM]` is better in my opinion, but I think you can have duplicates so both is best if you ask me.

**Review by Visne — CHANGES_REQUESTED** [review]

File: .github/keylabeler.yml (hunk 1 above)
Comment (DrSmugleaf) [reply .github/keylabeler.yml:13]: Done, both are valid now.

**Review by Visne — APPROVED** [review]

**Comment by Visne** [comment]
I think it would be a good idea to put a link to the keylabeler.yml file in the PR/issue templates, because it's still a little vague at the moment. For example, it's not very clear if you should type [TYPE: BUG] or [BUG], and not all labels are supported (like size).

**Comment by Peptide90** [comment]
Is there a reason all feature labels got nuked? I know it could be seen as clutter but it's also good for tagging related issues and features so if someone is working on medical, we can find related issues when they used obscure key words.

**Pull request merged**